userguide/storage.rst

```diff
@@ -1772,7 +1772,8 @@ management settings, acoustic level settings, and whether
 :ref:`S.M.A.R.T.` tests are enabled. The pool associated with the disk
 is displayed in the :guilabel:`Pool` column. *Unused* is displayed if
 the disk is not being used in a pool. Click :guilabel:`COLUMNS` to
-adjust the table.
+add or remove additional information as columns. Any information that
```

Comment:
This is kind of oddly worded.  The important part here is that the COLUMNS menu is for adding or removing columns shown by the table.  I'm not clear on what the last sentence is saying.